reflex/components/datadisplay/dataeditor.py

```diff
@@ -223,35 +276,6 @@ def _get_imports(self):
             },
         )
 
-    def get_event_triggers(self) -> Dict[str, Callable]:
```

Comment:
removing this has resulted in the addition of the base Component.get_event_triggers default triggers, is that appropriate for this component? or should we have a `get_event_triggers` here that returns an empty dict?

Reply:
Adding `get_event_triggers` that returns `{}` has caused events triggers to be completely removed from the docs.